src/main/java/frc/robot/subsystems/conveyor/Conveyor.java

```diff
@@ -32,6 +32,7 @@ public static void initialize(ConveyorIO io) {
     }
 
     public Command setVelocity(Supplier<MutableMeasure<Velocity<Angle>>> velocity) {
+        inputs.velocitySetpoint = velocity.get();
         return run(() -> io.setVelocity(velocity.get()));
```

Comment:
This function is called on the creation of the commands. Update the setpoint then would mean that the setpoint is updated only in robot init. Instead, use `runOnce` before the `run` function and set the setpoint.

Reply:
9dd7e865d61422e5653a9e28dddc43c7b23fdaf9